### PR TITLE
Date Format Option

### LINKED ISSUE #2: Problem with date format
I have a problem with the date format. The validator doesn't work for certain formats. I made a pull request with the changes. Thanks.

### PR DESCRIPTION
**Pull request by matias-yii**
I added the changes you asked. Please let me know if it's ok. Thanks

Fixes #2.

### CODE REVIEW

File: DateTimeCompareValidator.php [hunk 1]
```diff
@@ -154,10 +158,10 @@ protected function validateValue($value)
      * @param mixed $compareValue another value being compared
      * @return boolean whether the comparison using the specified operator is true.
      */
-    protected function compareValues($operator, $value, $compareValue)
+    protected function compareValues($operator, $value, $compareValue, $format)
     {
-    	$dateValue = new \DateTime($value);
-    	$dateCompareValue = new \DateTime($compareValue);
+        $dateValue = empty($format) ? \DateTime::createFromFormat($format, $value) : new \DateTime($value);
+        $dateCompareValue = empty($format) ? \DateTime::createFromFormat($format, $compareValue) : new \DateTime($compareValue);
```
Comment (EAnushan) [inline DateTimeCompareValidator.php]: You've got your conditionals here backwards. If `$format` is empty, you want to use `new \DateTime($value)`. Reverse the order.

File: DateTimeCompareValidator.php [hunk 2]
```diff
@@ -62,7 +62,11 @@ class DateTimeCompareValidator extends Validator
      * - `{compareAttribute}`: the label of the attribute to be compared with
      */
     public $message;
-
+    /**
+     * @var string Date format according to DateTime::createFromFormat specification.
+     *
```
Comment (EAnushan) [inline DateTimeCompareValidator.php]: You have an extra newline here. Try to have no new lines above or below the `@var` line. Take a look at the comments on other variables in this file.

Comment (matias-yii) [inline DateTimeCompareValidator.php]: I think that "-" means that the extra newline has been deleted. On the editor i don't see an extra new line there.

**Comment by matias-yii** [comment]
The changes are done. Please let me know if it's ok. Thanks .

File: DateTimeCompareValidator.php (hunk 2 above)
Comment (EAnushan) [inline DateTimeCompareValidator.php]: It's still there when I look at https://github.com/matias-yii/yii2-datetime-compare/blob/master/DateTimeCompareValidator.php

Maybe your editor is being nice and hiding it?

**Comment by EAnushan** [comment]
Everything looks good other than the newline now. Might be your editor.

**Comment by matias-yii** [comment]
Please see the screenshot below or check the raw code on github I don't see the extra new line.
screenshot - 14_12_2015 04_34_06 p_m

**Comment by EAnushan** [comment]
The third last line in your screenshot, just about `$format` has a newline doesn't it?

**Comment by matias-yii** [comment]
Ah ok, that line, sorry I misunderstood you. Done. Thanks.

**Comment by EAnushan** [comment]
Thank you! :) I merged your pull request and tagged a new release. You should be able to grab version 2.1.0 from packagist shortly.

**Comment by matias-yii** [comment]
Great Work. Thanks!!! :)

**Pull request merged**